buddyboss/buddyboss-platform
# 1059 - Fixed the translation issue

### All Submissions:






### Changes proposed in this Pull Request:



Fixes https://github.com/buddyboss/buddyboss-platform/issues/1059

### How to test the changes in this Pull Request:

1. Switch to another site language with prebuilt translation such as German/French/Hungarian.
2. Using Loco Translate, in Plugins > BuddyBoss Platform, translate the "By creating an account you are agreeing to the %s and %s." string.
3. Go to the registration page and notice it will not be translated.

### Proof Screenshots or Video



### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
